cmd.sh

```diff
@@ -14,8 +14,17 @@ set -x
 # this is to prevent accidental uploads
 echo "Getting anaconda token from github secrets..."
 
-ANACONDA_ORG="scientific-python-nightly-wheels"
+ANACONDA_ORG="${INPUT_ANACONDA_NIGHTLY_UPLOAD_ORGANIZATION}"
```

Comment:
would this break the current behaviour for the core libraries uploading to the default location?

Reply:
No, because I added the default location in this line: https://github.com/scientific-python/upload-nightly-action/pull/47/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R20

However, I believe that:

1. this parameter should be required instead of optional  
2. After merging, we should ask the core libraries to add and update this variable
3. Then, when it is done, create a new PR here to remove this default location and let it empty.

Thank you for your review @bsipocz!

Reply:
@skoudoro While I'm not against the idea of being explicit about the target channel (I think that's a good idea in general to guard against default changes), I think the original intent for the GitHub Action was to make uploading as easy as possible for the core packages with the idea that they need to configure things as minimally as possible.

I think that this could probably be discussed more broadly outside of this PR though, so maybe you can open up an Issue from your comment to discuss what the target audience and scope of the action configuration should be?